### ISSUE TITLE
Titanic - Back-Propagation in Numpy

### ISSUE THREAD

**Issue by Aakriti28** [wncc/Hello-FOSS-ML#11]
A neural network has been coded from scratch in Numpy on the dataset in [this notebook](https://github.com/wncc/Hello-FOSS-ML/blob/master/Titanic%20neural%20network%20Numpy%20from%20scratch.ipynb) but it is incomplete; it misses the function for calculating gradients during back-propagation. Your task is to complete the function for calculating gradients, train the model and output the predictions on test data to a csv file.

**Comment by sheelfshah** [comment]
I have taken this up... #31
